src/components/common/RouterLinkNetwork.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <router-link
+    :class="cclass"
```

Comment:
this is very strange

Reply:
Could you explain? We can't use class property there

Reply:
i guess i would rather use a different word or change it with a symbol or something. i'm not sure. 

`:class="class_"` 

all feels strange.

Reply:
I changed it for `routerClass`. What do you think?

Reply:
I don't know why i have chosen cclass for this variable, so any changes will be good)